### PR TITLE
Code conventions

### PR DESCRIPTION
**Pull request by JorgeRCV**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 1]
```diff
@@ -36,27 +33,39 @@ public void audioConverter() {
         String hz = parameter.formatHz();
         String volume = parameter.formatVolume();
         String audioChannel = parameter.formatAudioChannel();
-        String input = parameter.getFilePath().substring(parameter.getFilePath().lastIndexOf(System.getProperty("file.separator")) + 1);
-        setOutputFileName(parameter.getFilePath().substring((parameter.getFilePath().lastIndexOf(System.getProperty("file.separator")) + 1),
-                parameter.getFilePath().lastIndexOf(".") + 1) + parameter.getFormat());
+        String input = parameter.getFilePath()
+                       .substring(parameter.getFilePath()
+                       .lastIndexOf(System.getProperty("file.separator")) + 1);
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java]: these lines should be in one

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 2]
```diff
@@ -36,27 +33,39 @@ public void audioConverter() {
...
+        setOutputFileName(parameter.getFilePath()
+                         .substring((parameter.getFilePath()
+                         .lastIndexOf(System.getProperty("file.separator")) + 1),
+                         parameter.getFilePath().lastIndexOf(".") + 1) + parameter.getFormat());
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java]: these lines should be in one

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 3]
```diff
@@ -36,27 +33,39 @@ public void audioConverter() {
...
-                + overwrite;
+        String command = relativePath + ffmpeg +
+                         input +
+                         audioChannel +
+                         bitrate +
+                         hz +
+                         volume +
+                         getOutputFileName() +
+                         overwrite;
         System.out.println(command);
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java]: these lines should be in one

File: src/main/java/org/fundacion/jala/converter/service/ChecksumService.java [hunk 4]
```diff
@@ -1,6 +1,6 @@
 /**
  * Copyright (c) 2021 Fundacion Jala.
- *
+ * <p>
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/ChecksumService.java]: This tag <p> should be deleted

File: src/main/java/org/fundacion/jala/converter/service/ChecksumService.java [hunk 5]
```diff
@@ -29,12 +29,13 @@
...
-     * @throws IOException when invalid file's path
-     * @throws NoSuchAlgorithmException when invalid algorithm is provided
+     * @param filePath the file's direction.
+     * @return a String with the checksum.
+     * @throws IOException when invalid file's path.
+     * @throws NoSuchAlgorithmException when invalid algorithm is provided.
      */
-    public static String getFileChecksum(final String filePath) throws IOException, NoSuchAlgorithmException {
+    public static String getFileChecksum(final String filePath)
+           throws IOException, NoSuchAlgorithmException {
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/ChecksumService.java]: these lines should be in one

File: src/main/java/org/fundacion/jala/converter/service/ExtractMetadata.java [hunk 6]
```diff
@@ -23,24 +28,28 @@
...
 
-    public ExtractMetadata(ObjectMetadata extractMetadata) {
+    public ExtractMetadata(final ObjectMetadata extractMetadata) {
         this.fileToExtract = extractMetadata.getFileToExtract();
         if (extractMetadata.getMoreInfo()) setMoreInformation();
-        exportTypeFile = new ExportTypeFile(fileToExtract.getName(), extractMetadata.getNameExport(), extractMetadata.getTypeFileExport(), extractMetadata.getFileToExport());
+        exportTypeFile = new ExportTypeFile(fileToExtract.getName(),
+                         extractMetadata.getNameExport(),
+                         extractMetadata.getTypeFileExport(),
+                         extractMetadata.getFileToExport());
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/ExtractMetadata.java]: Maybe these lines should be in two lines

File: src/main/java/org/fundacion/jala/converter/service/ExtractMetadata.java [hunk 7]
```diff
@@ -23,24 +28,28 @@
...
         exportFile = exportTypeFile.getNameFileCompleteToExport();
         extractMetadata();
     }
 
     public ExtractMetadata(final File fileExtract, final File fileExport) {
         this.fileToExtract = fileExtract;
         setMoreInformation();
-        exportTypeFile = new ExportTypeFile(fileToExtract.getName(), "Default", TypeFileExport.TXT, fileExport);
+        exportTypeFile = new ExportTypeFile(fileToExtract.getName(),
+               "Default", TypeFileExport.TXT, fileExport);
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/ExtractMetadata.java]: Maybe these lines should be in one

File: src/main/java/org/fundacion/jala/converter/service/ExtractMetadata.java [hunk 8]
```diff
@@ -56,24 +65,28 @@ public void extractMetadata() {
...
+     * Extracts metadata from a file.
+     *
+     * @param metadata a String with metadata request.
+     * @param outputFileName the new file's name.
+     * @param fileStorageService object to create the path.
      */
-    public static void extractMetadata(final String metadata, final String outputFileName, final FileStorageService fileStorageService) {
+    public static void extractMetadata(final String metadata,
+                                       final String outputFileName,
+                                       final FileStorageService fileStorageService) {
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/ExtractMetadata.java]: these lines should be in one or two lines

File: src/main/java/org/fundacion/jala/converter/service/ExtractText.java [hunk 9]
```diff
@@ -13,60 +15,68 @@
...
     }
+
     public ExtractText(final String language, final String pathFile) {
         this.language = language;
         this.pathFile = pathFile;
     }
 
-    public ExtractText(final String language, final String pathFile, final String nameOutputFile) {
+    public ExtractText(final String language, final String pathFile,
+                       final String nameOutputFile) {
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/ExtractText.java]: these lines should be in one

File: src/main/java/org/fundacion/jala/converter/service/ExtractText.java [hunk 10]
```diff
@@ -13,60 +15,68 @@
...
      */
     public String getPathFile() {
         return pathFile;
     }
 
     /**
+     * Returns the output name.
      *
-     * @return a string with the name of file generate
+     * @return a string with the name of file generate.
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/ExtractText.java]: in returns the word string should be capitalized String

File: src/main/java/org/fundacion/jala/converter/service/PythonCompiler.java [hunk 11]
```diff
@@ -5,36 +5,41 @@
...
      */
     public String compiler(final PythonParameter newPythonParameter) {
         LOGGER.info("start");
         try {
             this.pythonParameter = newPythonParameter;
             LOGGER.info("Execute Try");
-            String command = pythonParameter.getPythonEnum().getVersion() + " " + "\"" + pythonParameter.getFilePath()+ "\"";
+            String command = pythonParameter.getPythonEnum().getVersion() + " " + "\"" +
+                             pythonParameter.getFilePath() + "\"";
             Process process = Runtime.getRuntime().exec(command);
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/PythonCompiler.java]: these lines should be in one

File: src/main/java/org/fundacion/jala/converter/service/VideoConverter.java [hunk 12]
```diff
@@ -37,12 +40,13 @@ public VideoConverter(final VideoParameter videoParameter) {
     public void convertVideo() {
         String adaptPath = "\"" + parameter.getFilePath() + "\"";
         format = parameter.getOutputFormat();
-        output = adaptPath.substring((adaptPath.lastIndexOf("\\") + 1), adaptPath.lastIndexOf(".") + 1) + format;
+        output = adaptPath.substring((adaptPath.lastIndexOf("\\") + 1),
+                 adaptPath.lastIndexOf(".") + 1) + format;
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/VideoConverter.java]: these lines should be in one

File: src/main/java/org/fundacion/jala/converter/view/compiler/CompilerInterface.java [hunk 13]
```diff
@@ -66,8 +65,9 @@ public CompilerInterface(final String newToken) {
      * @param gw represents width grid.
      * @return a custom GridBagConstraints.
      */
-    public GridBagConstraints setConstraints(final GridBagConstraints panelConstraint, final int gx, final int gy,
-                                            final int gh, final int gw) {
+    public GridBagConstraints setConstraints(final GridBagConstraints panelConstraint,
+                                             final int gx, final int gy,
+                                             final int gh, final int gw) {
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/compiler/CompilerInterface.java]: these lines should be in one or two lines

File: src/main/java/org/fundacion/jala/converter/view/metadata/MetaDataInterface.java [hunk 14]
```diff
@@ -75,6 +76,7 @@ public MetaDataInterface(final String newToken) {
     /**
      * Converts, sends information for metadataCLASS conversion.
      * Shows a Dialog with the information.
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/metadata/MetaDataInterface.java]: the description of the methods should only be on one line

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 15]
```diff
@@ -36,27 +33,39 @@ public void audioConverter() {
...
     public void setOutputFileName(final String newOutputFileName) {
         this.outputFileName = newOutputFileName;
     }
 
     /**
-     * Gets name of the output filename
-     * @return a string of output filename
+     * Gets name of the output filename.
+     *
+     * @return a string of output filename.
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java]: string should be capitalized String

File: src/main/java/org/fundacion/jala/converter/service/ChecksumService.java [hunk 16]
```diff
@@ -56,11 +57,11 @@ public static String getFileChecksum(final String filePath) throws IOException,
...
-     * @throws IOException when invalid path
-     * @throws NoSuchAlgorithmException when invalid algorithm provided
+     * @param firstFilePath first file's path.
+     * @param checksum a String with the checksum.
+     * @return a Boolean with the response.
+     * @throws IOException when invalid path.
+     * @throws NoSuchAlgorithmException when invalid algorithm provided.
      */
     private static Boolean repeatedChecksum(final String firstFilePath,
                                             final String checksum)
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/ChecksumService.java]: these lines should be in one

File: src/main/java/org/fundacion/jala/converter/service/ExtractMetadata.java [hunk 17]
```diff
@@ -56,24 +65,28 @@ public void extractMetadata() {
...
         String outputPath = fileStorageService.getOutputPath(outputFileName);
         String outputPathWithoutFileName = fileStorageService.getOutputPathWithoutFileName(outputFileName);
         if (metadata.equals("true")) {
-            ExtractMetadata extractMetadata = new ExtractMetadata(new File(outputPath), new File(outputPathWithoutFileName));
+            ExtractMetadata extractMetadata = new ExtractMetadata(new File(outputPath),
+                                              new File(outputPathWithoutFileName));
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/ExtractMetadata.java]: these lines should be in one

File: src/main/java/org/fundacion/jala/converter/service/ExtractText.java [hunk 18]
```diff
@@ -102,8 +113,9 @@ private void safeInfo(final String name, final String text) {
         PrintWriter printWriter;
         try {
             LOGGER.info("Execute Try");
-            file = new File(System.getProperty("user.dir") + "\\archive\\" + name + ".txt");
-            fileWriter =  new FileWriter(file);
+            file = new File(System.getProperty("user.dir") +
+                            "\\archive\\" + name + ".txt");
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/ExtractText.java]: These lines should be in one

File: src/main/java/org/fundacion/jala/converter/service/VideoConverter.java [hunk 19]
```diff
@@ -105,7 +111,8 @@ private void generateAThumbnail() {
 
     /**
      * Removes the audio of the input video.
-     * @return audio command
+     *
+     * @return audio command.
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/VideoConverter.java]: the description of all returns should be more descriptive

File: src/main/java/org/fundacion/jala/converter/service/ZipService.java [hunk 20]
```diff
@@ -27,11 +27,12 @@
...
-     * @param filePath the file's direction
-     * @param zipOutputStream an output stream filter to write the zip file
-     * @throws IOException when invalid path is given
+     * @param filePath the file's direction.
+     * @param zipOutputStream an output stream filter to write the zip file.
+     * @throws IOException when invalid path is given.
      */
-    public static void zipProcess(final String filePath, final ZipOutputStream zipOutputStream) throws IOException {
+    public static void zipProcess(final String filePath, final ZipOutputStream zipOutputStream)
+           throws IOException {
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/ZipService.java]: these lines should be in one

File: src/main/java/org/fundacion/jala/converter/service/ZipService.java [hunk 21]
```diff
@@ -47,11 +48,12 @@ public static void zipProcess(final String filePath, final ZipOutputStream zipOu
...
-     * @param filePath the file's direction
-     * @param outputPath the destination's direction
-     * @throws IOException when invalid path is given
+     * @param filePath the file's direction.
+     * @param outputPath the destination's direction.
+     * @throws IOException when invalid path is given.
      */
-    public static void zipFile(final String filePath, final String outputPath) throws IOException {
+    public static void zipFile(final String filePath, final String outputPath)
+           throws IOException {
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/ZipService.java]: These lines should be in one

File: src/main/java/org/fundacion/jala/converter/view/compiler/LanguageButtons.java [hunk 22]
```diff
@@ -43,18 +43,18 @@ protected LanguageButtons() {
     }
 
     /**
-     * Button that defines or not java is selected
+     * Button that defines is java is selected or not.
      *
-     * @return a JButton
+     * @return a JButton.
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/compiler/LanguageButtons.java]: returns should be more descriptive and describe the type of data they return

File: src/main/java/org/fundacion/jala/converter/view/metadata/MetaDataInterface.java [hunk 23]
```diff
@@ -50,6 +50,7 @@
 
     /**
      * Initializes the graphics components for MetaData interface.
+     *
      * @param newToken a String with the authentication token.
      */
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/metadata/MetaDataInterface.java]: Constructor methods should not have javadoc

File: src/main/java/org/fundacion/jala/converter/view/metadata/MetaDataInterface.java [hunk 24]
```diff
@@ -99,6 +101,7 @@ public void actionPerformed(final ActionEvent e) {
 
     /**
      * Obtains the request
+     *
      * @throws IOException
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/metadata/MetaDataInterface.java]: Description of the exception is missing

File: src/main/java/org/fundacion/jala/converter/view/metadata/OutputInfo.java [hunk 25]
```diff
@@ -67,6 +67,7 @@ protected OutputInfo() {
 
     /**
      * Gets the name of the outputfile.
+     *
      * @return outputname.
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/metadata/OutputInfo.java]: Return should be more descriptive

File: src/main/java/org/fundacion/jala/converter/view/metadata/MetaDataInterface.java (hunk 24 above)
Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/view/metadata/MetaDataInterface.java]: Is missing a description of exception

Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/view/metadata/MetaDataInterface.java]: then is missing "."  a the end of the javafoc

File: src/main/java/org/fundacion/jala/converter/service/FileStorageService.java [hunk 26]
```diff
@@ -53,19 +66,30 @@ public Resource downloadFile(String fileName) {
...
     public static String getOutputPath(String filename) {
         return PATH + File.separator + ARCHIVE + File.separator + File.separator + filename;
     }
+
     /**
-     * Returns the designed output path without the file's name
+     * Returns the designed output path without the file's name.
+     *
+     * @param filename to get an output path.
+     * @return path withour the file name.
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/service/FileStorageService.java]: the word "withour" is wrong I think that should be "without"

File: src/main/java/org/fundacion/jala/converter/service/VideoConverter.java [hunk 27]
```diff
@@ -119,7 +126,8 @@ private String removeAudio() {
 
     /**
      * Changes the input video frame rate.
-     * @return frame command
+     *
+     * @return frame command.
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/service/VideoConverter.java]: the description of all returns should be more descriptive

File: src/main/java/org/fundacion/jala/converter/service/VideoConverter.java [hunk 28]
```diff
@@ -133,15 +141,17 @@ private String changeFrameRate() {
...
      */
     public void setOutputFileName(final String newOutputFileName) {
         this.outputFileName = newOutputFileName;
     }
 
     /**
      * Gets the output file name.
-     * @return outputFileName
+     *
+     * @return outputFileName.
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/service/VideoConverter.java]: the description of all returns should be more descriptive

File: src/main/java/org/fundacion/jala/converter/view/compiler/CompilerButtons.java [hunk 29]
```diff
@@ -55,7 +55,7 @@ public void actionPerformed(final ActionEvent e) {
      * @return a JButton.
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/view/compiler/CompilerButtons.java]: Is miss a space empty between line 54 and 55

then the description of all returns should be more descriptive

Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/view/compiler/CompilerButtons.java]: I think is missing a space empty between line 46 and 47 too in that class

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 30]
```diff
@@ -36,27 +33,31 @@ public void audioConverter() {
...
-                parameter.getFilePath().lastIndexOf(".") + 1) + parameter.getFormat());
+        String input = parameter.getFilePath().substring(parameter.getFilePath()
+                       .lastIndexOf(System.getProperty("file.separator")) + 1);
+        setOutputFileName(parameter.getFilePath().substring((parameter.getFilePath()
+                         .lastIndexOf(System.getProperty("file.separator")) + 1),
+                         parameter.getFilePath().lastIndexOf(".") + 1) + parameter.getFormat());
         String overwrite = " -y";
-        String command = relativePath + ffmpeg + input + audioChannel + bitrate + hz + volume + getOutputFileName()
-                + overwrite;
+        String command = relativePath + ffmpeg + input + audioChannel + bitrate + hz + volume + getOutputFileName() +
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java]: the plus sign must start on the other line "+ overwrite;"

File: src/main/java/org/fundacion/jala/converter/service/metadata/ExportTypeFile.java [hunk 31]
```diff
@@ -96,28 +96,25 @@ private void setNameFileExport(final String newName) {
...
     private void exportFileTxt() {
-        nameFileCompleteToExport = " > " + "\"" +
-                addressExportFile.getAbsolutePath() + "\\" +
-                nameFileExport + ".txt" + "\"";
+        nameFileCompleteToExport = " > " + "\"" + addressExportFile.getAbsolutePath() + "\\" +
+                                   nameFileExport + ".txt" + "\"";
     }
 
     /**
      * Defines file are exported with type Html.
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/service/metadata/ExportTypeFile.java]: The javadoc should be in one line

File: src/main/java/org/fundacion/jala/converter/service/metadata/ExportTypeFile.java [hunk 32]
```diff
@@ -96,28 +96,25 @@ private void setNameFileExport(final String newName) {
...
     private void exportFileHtml() {
-        nameFileCompleteToExport = " -h > " + "\"" +
-                addressExportFile.getAbsolutePath() + "\\" +
-                nameFileExport + ".html" + "\"";
+        nameFileCompleteToExport = " -h > " + "\"" + addressExportFile.getAbsolutePath() + "\\" +
+                                   nameFileExport + ".html" + "\"";
     }
 
     /**
      * Defines file are exported with type XMP.
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/service/metadata/ExportTypeFile.java]: The javadoc should be in one line

File: src/main/java/org/fundacion/jala/converter/service/VideoConverter.java [hunk 33]
```diff
@@ -76,7 +80,8 @@ private String changeResolution() {
         if (width > 0 && height > 0) {
             scale = width + ":" + height;
             aspectRatio = ":force_original_aspect_ratio=decrease,pad=";
-            resolutionCommand = "-vf \"scale=" + scale + aspectRatio + scale + ":-1:-1:color=white\"";
+            resolutionCommand = "-vf \"scale=" + scale + aspectRatio + scale +
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/VideoConverter.java:83]: the plus sign must start on the other line "+ overwrite;"

File: src/main/java/org/fundacion/jala/converter/service/VideoConverter.java [hunk 34]
```diff
@@ -88,10 +93,8 @@ private String changeResolution() {
     private void generateAThumbnail() {
         String name = getOutputFileName().substring(0, getOutputFileName().lastIndexOf("."));
         String startCommand = "ffmpeg -i ";
-        String outputCommand = pathOutput + output + "\"" + " -ss 00:00:01 -vframes 1 -s 128x128 "
-                + pathOutput
-                + name
-                + ".png\"";
+        String outputCommand = pathOutput + output + "\"" + " -ss 00:00:01 -vframes 1 -s 128x128 " +
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/VideoConverter.java:96]: the plus sign must start on the other line "+ overwrite;"

File: src/main/java/org/fundacion/jala/converter/service/VideoConverter.java (hunk 33 above)
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/VideoConverter.java:83]: the plus sign must start on the other line "+  ":-1:-1:color=white\"";

**Review by jpgonzales27 — APPROVED** [review]

**Review by cristianfchq — APPROVED** [review]

**Pull request merged**